src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -117,7 +117,7 @@ Just set the `pinned` prop.
 			'app-navigation-entry--collapsible': collapsible,
 		}">
 		<!-- Navigation item, can be either an <li> or a <router-link> depending on the props -->
-		<nav-element v-bind="navElement"
+		<div v-bind="navElement"
```

Comment:
I would leave it as `nav-element` to keep it obvious that it's for navigation. You can make it render as a div via the `navElement` object.

Reply:
and have it both 'div'?

Reply:
I would think so, yes.